Atomic/syscontainers.py

```diff
@@ -634,34 +634,54 @@ def _resolve_remote_path(self, remote_path):
             raise ValueError("The container's rootfs is set to remote, but the remote rootfs does not exist")
         return real_path
 
-    def _checkout(self, repo, name, img, deployment, mode, values=None, destination=None, extract_only=False, remote=None, prefix=None, installed_files_checksum=None, system_package='no'):
-        destination = destination or os.path.join(self._get_system_checkout_path(), "{}.{}".format(name, deployment))
-        unitfileout, tmpfilesout = self._get_systemd_destination_files(name, prefix)
+    def _checkout_wrapper(self, repo, name, img, deployment, mode, values=None, destination=None, extract_only=False, remote=None, prefix=None, installed_files_checksum=None, system_package='no'):
+        """
+        Wrapper function that groups parameters into a dictionary for better readbility
+
+        returns the same result that _checkout returns
+        """
+        options = {"name" : name,
+                   "img" : img,
+                   "deployment" : deployment,
+                   "UpgradeMode" : mode,
+                   "values" : values,
+                   "destination" : destination,
+                   "extract_only" : extract_only,
+                   "remote" : remote,
+                   "prefix" : prefix,
+                   "InstalledFilesChecksum" : installed_files_checksum,
```

Comment:
some options names use camel case, other ones use underscores.  Could we make them all in the underscore notation (i.e. replace UpgradeMode and InstalledFilesChecksum)?

Reply:
sounds good, sure, will do that